addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2142,8 +2143,9 @@ const ListUserValueWidget = UserValueWidget.extend({
     },
     /**
      * @private
+     * @param {Event} [ev] event that requires this notifyCurrentState.
```

Comment:
This param is no longer used since your last changes. It should be "preview" now.